client/src/resources/lang/en_us.json

```diff
@@ -1035,6 +1044,11 @@
     "wiki.article.role.werewolf.attributes":"* You track each player you have the scent of. You are told who each player you are tracking visits.\n* You still track the scent of players after you die\n* While you are in jail, you attack all Jailors who do not choose to execute you\n* You have an armor-piercing attack\n",
     "wiki.article.role.werewolf.extra":"* You will attack yourself if you visit yourself\n* You will not rampage the jailor while in jail\n* You will still attack or gain the scent of visits to you when you are in jail\n* You will still track the players you have the scent of while you are in jail\n* A roleblock has no effect on you besides erasing your visits. If you are roleblocked, it is as if you visited nobody\n* Players attacked by a rampage are indirectly attacked",
 
+    "wiki.article.role.ojo.guide":"- You may choose to either kill every player with a certain role, or find out every player with a certain role\n\n_For Example_\n- You choose to kill all lookouts and then learn that @1 and @2 are lookouts, and then they die\n- You choose to see all trackers, and you see nobody, so you know there is no tracker",
+    "wiki.article.role.ojo.abilities":"- You may choose to do one of the following\n    - See all players with a certain role, you visit all players with that role\n    - Kill all players with a certain role, you visit all players with that role with an attacking visit\n        - Players you visit with an attacking visit will be attacked with an armor-piercing attack",
+    "wiki.article.role.ojo.attributes":"- You are told the name and role of all players you visit, weather you choose see or kill\n- If you are dead, you can not choose to see or kill",
+    "wiki.article.role.ojo.extra":"- Your ability will work on the person you actually visited, so if a transporter transports a player you visit with someone else, you will see or attack the other player\n- A Witch can force you to visit someone, if that happens, you wont attack, because the witch doesn't force you to use an attacking visit\n- You don't visit yourself\n- You don't visit dead players",

```

Comment:
In this case, do you get their actual role, or does it tell you that they are the role you investigated?